superset/reports/commands/execute.py

```diff
@@ -94,35 +94,39 @@ def __init__(
         self._start_dttm = datetime.utcnow()
         self._execution_id = execution_id
 
-    def set_state_and_log(
+    def update_report_schedule_and_log(
```

Comment:
I felt that `update_report_schedule_and_log` was more reflective of the function logic.